src/huggingface_hub/repocard.py

```diff
@@ -572,13 +573,16 @@ def metadata_eval_result(
             The name of the metric configuration used in `load_metric()`.
             Example: bleurt-large-512 in `load_metric("bleurt", "bleurt-large-512")`.
         metrics_verified (`bool`, *optional*, defaults to `False`):
-            If true, indicates that evaluation was generated by Hugging Face (vs. self-reported).
-            If a user tries to push self-reported metric results with verified=True, the push
-            will be rejected.
+            Automatically computed, do not set. Dynamically overridden by
```

Comment:
I took this wording from [here](https://github.com/huggingface/moon-landing/pull/4037/files#diff-e9eafbf94fe033bdaf22611fa3eab89f02aa8776685889dd5cace85d7ed82257R76). Happy to reword if needed